pom.xml

```diff
@@ -85,7 +85,7 @@
         <hamcrest.version>1.3</hamcrest.version>
         <mockito.version>1.10.19</mockito.version>
         <powermock.version>1.6.6</powermock.version>
-        <jmh.version>1.16</jmh.version>
+        <jmh.version>1.21</jmh.version>
```

Comment:
This was tried before - https://github.com/hazelcast/hazelcast/pull/12355

Reply:
Thanks for the reference. Reverted.